src/Data/String/LineWrapping.purs

```diff
@@ -106,5 +114,5 @@ printWrappedLine =
     SingleWordWrappedLine x →
       printMeasuredWord x
     MultipleWordWrappedLine x1 x2 xs →
-      String.joinWith " " $ printMeasuredWord <$> [x1] <> [x2] <> xs
+      String.joinWith "\n" $ printMeasuredWord <$> [x1] <> [x2] <> xs
```

Comment:
you're quite right, i've removed these constructors in favour of an array. @cmdv and i wrote a test to check that the first word of a line won't fit on the previous line too.